app/src/main/java/com/example/mylittlelibrary/ui/activities/MainActivity.kt

```diff
@@ -42,27 +42,27 @@ class MainActivity : AppCompatActivity() {
             intent.putExtra("Clicked", "DVDs")
             startActivity(intent)
 
-        imageView = findViewById(R.id.imageView)
-        button = findViewById(R.id.btnAddItem)
+            imageView = findViewById(R.id.imageView)
+            button = findViewById(R.id.btnAddItem)
 
-        button.setOnClickListener{
-            val takePictureIntent = Intent(MediaStore.ACTION_IMAGE_CAPTURE)
+            button.setOnClickListener{
+                val takePictureIntent = Intent(MediaStore.ACTION_IMAGE_CAPTURE)
 
-            try {
-                startActivityForResult(takePictureIntent, REQUEST_IMAGE_CAPTURE)
-            } catch (e:ActivityNotFoundException){
-                // Toast.makeText(this,"Error: " + e.LocalizedMessage,Toast.LENGTH_SHORT).show()
-                Toast.makeText(this,"Error",Toast.LENGTH_SHORT).show()
+                try {
+                    startActivityForResult(takePictureIntent, REQUEST_IMAGE_CAPTURE)
+                } catch (e:ActivityNotFoundException){
+                    // Toast.makeText(this,"Error: " + e.LocalizedMessage,Toast.LENGTH_SHORT).show()
+                    Toast.makeText(this,"Error",Toast.LENGTH_SHORT).show()
+                }
             }
-        }
-        fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
-            if (requestCode == REQUEST_IMAGE_CAPTURE && resultCode == RESULT_OK) {
-                val imageBitmap = data?.extras?.get("data") as Bitmap
-                imageView.setImageBitmap(imageBitmap)
-            } else {
-                super.onActivityResult(requestCode, resultCode, data)
+            fun onActivityResult(requestCode: Int, resultCode: Int, data: Intent?) {
+                if (requestCode == REQUEST_IMAGE_CAPTURE && resultCode == RESULT_OK) {
+                    val imageBitmap = data?.extras?.get("data") as Bitmap
+                    imageView.setImageBitmap(imageBitmap)
+                } else {
+                    super.onActivityResult(requestCode, resultCode, data)
+                }
```

Comment:
Many lines of code inside the binding.dvds.setOnClickListener curly brackets were formatted to be on the same column of code as binding.dvds.setOnClickListener and it looked like those lines of code were outside the setOnClickListener curly brackets. The change to shift the lines of code would make the code fit more with kotlinlang.org's coding conventions for indentation.
https://kotlinlang.org/docs/coding-conventions.html#indentation